packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -468,11 +468,32 @@
     background: none !important; /* stylelint-disable-line declaration-no-important */
   }
 
+  + .bpk-checkbox__icon {
```

Comment:
This class shouldn't exist anywhere in `bpk-mixins` package - can it not be in the components scss file?

Reply:
Yeah fair it should be possible to move it